manifests/0000_70_dns-operator_02-deployment.yaml

```diff
@@ -37,4 +37,13 @@ spec:
             requests:
               cpu: 10m
       tolerations:
-      - operator: Exists # dns operator should be schedulable always.
+      - key: "node-role.kubernetes.io/master"
+        operator: "Exists"
+        effect: "NoSchedule"
+      - key: "node.kubernetes.io/unreachable"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      - key: "node.kubernetes.io/not-ready"
+        operator: "Exists"
+        effect: "NoExecute"
```

Comment:
This should be `NoSchedule` too, if we want this operator to land on node which is `not-Ready`.